docker/ubuntu/baseimage-18.04/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM phusion/baseimage:bionic-1.0.0
+ARG REGISTRY=docker.io
+FROM $REGISTRY/phusion/baseimage:bionic-1.0.0
```

Comment:
This one doesn't actually exist on ghcr.io because it's too old, yet being different would be confusing too.